packages/trader/src/Modules/Trading/Components/Form/ContractType/ContractTypeInfo/contract-type-info.jsx

```diff
@@ -21,6 +21,7 @@ const Info = ({ handleNavigationClick, handleSelect, initial_index, item, list }
         i => i.value !== 'rise_fall_equal' && i.value !== 'turbosshort'
     );
     const has_toggle_buttons = /accumulator|vanilla/i.test(carousel_index);
+    const should_show_video = /accumulator|vanilla/i.test(carousel_index);
```

Comment:
@kate-deriv should_show_video and has_toggle_buttons have the same value. can we use 1 variable which would unite this 2 concepts? like `has_toggle_and_video`

Reply:
We can, but those variables won't be qual all the time. Example: Turbos was released and after content changes it will have a toggle button, but won't a video. So 
```
const has_toggle_buttons = /accumulator|vanilla|turbos/i.test(carousel_index);
const should_show_video = /accumulator|vanilla/i.test(carousel_index);
```